cmd/account/reset.go

```diff
@@ -120,19 +124,30 @@ func (o *resetOptions) run() error {
 	account.Spec.ClaimLink = ""
 	account.Spec.ClaimLinkNamespace = ""
 	account.Spec.IAMUserSecret = ""
+	if o.resetLegalEntity {
+		account.Spec.LegalEntity = v1alpha1.LegalEntity{}
+	}
+
 	if err := o.kubeCli.Update(ctx, account, &client.UpdateOptions{}); err != nil {
 		return err
 	}
 
 	// reset fields in status
 	var mergePatch []byte
+
+	status := map[string]interface{}{
```

Comment:
I believe it would make the code cleaner if you could convert the map to an anonymous struct.

Reply:
I'll leave it the way it currently is - thanks